lib/active_model/serializer.rb

```diff
@@ -26,35 +26,12 @@ class << self
     end
 
     def self.inherited(base)
-      base._attributes = self._attributes.try(:dup) || []
-      base._attributes_keys = self._attributes_keys.try(:dup) || {}
```

Comment:
Good point, I think it's better not to use `try` if we can make code safe without it.